test/io.test.mjs

```diff
@@ -0,0 +1,61 @@
+import { createAntAosLoader } from './utils.mjs';
+import { describe, it } from 'node:test';
+import assert from 'node:assert';
+import {
+  AO_LOADER_HANDLER_ENV,
+  DEFAULT_HANDLE_OPTIONS,
+} from '../tools/constants.mjs';
+
+describe('IO Network Updates', async () => {
+  const { handle: originalHandle, memory: startMemory } =
+    await createAntAosLoader();
+
+  async function handle(options = {}, mem = startMemory) {
+    return originalHandle(
+      mem,
+      {
+        ...DEFAULT_HANDLE_OPTIONS,
+        ...options,
+      },
+      AO_LOADER_HANDLER_ENV,
+    );
+  }
+
+  it('should send update to io network when a name is released', async () => {
+    const result = await handle({
+      Tags: [
+        { name: 'Action', value: 'Release-Name' },
+        { name: 'IO-Process-Id', value: 'io-process-id' },
+        { name: 'Name', value: 'name' },
+      ],
+    });
+
+    // two messages should be sent - one to the io process and one to the sender
```

Comment:
Consider ensuring that both messages were sent and to the intended recipients.

Reply:
the integration tests validate that both messages were sent. will add assertions on the target addresses of the messages

Reply:
[b07e874](https://github.com/ar-io/ar-io-ant-process/pull/23/commits/b07e874334d377ba35bb2056c4b9dbdd974a976f)